OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -312,7 +312,8 @@ public MovePart(Move move, WPos from, WPos to, int fromFacing, int toFacing, int
 				MoveFractionTotal = (to - from).Length;
 
 				// Calculate an elliptical arc that joins from and to
-				if (fromFacing != toFacing)
+				var delta = Util.NormalizeFacing(fromFacing - toFacing);
+				if (delta != 0 && delta != 128)
```

Comment:
Oh wait, no. Still, this doesn't catch it.

![pillanatfelvetel1](https://cloud.githubusercontent.com/assets/1136302/18034342/8140db5a-6d3a-11e6-884e-7061b202bccb.png)